dts/bindings/spi/spi-device.yaml

```
@@ -58,3 +61,22 @@ properties:
       without releasing it. A typical use case is variable length SPI packets
       where the first spi transaction reads the length and the second spi transaction
       reads length bytes.
+  spi-cs-high:
+    type: boolean
+    description: The device requires the chip select to be active high.
+  spi-interframe-delay-ns:
+    type: int
+    default: 0
+    description: |
+      Delay in nanoseconds between SPI words.
+      Default of 0 is special value which will assume to mean half of the SCK period.
+  spi-cs-setup-delay-ns:
+    type: int
+    description: |
+      Delay in nanoseconds to be introduced by the controller after CS is asserted.
+      Also known as enable lead time.
+  spi-cs-hold-delay-ns:
+    type: int
+    description: |
+     Delay in nanoseconds to be introduced by the controller before the CS is de-asserted.
+      Also known as enable lag time.
```

Non blocking
```suggestion
     Also known as enable lag time.
```
Also, maybe (still non blocking):
```suggestion
     Also known as "enable lag time".
```
To be applied above as well.